leader-election-impl/src/main/java/com/palantir/paxos/SqlitePaxosStateLog.java

```diff
@@ -30,26 +30,34 @@
 
 public final class SqlitePaxosStateLog<V extends Persistable & Versionable> implements PaxosStateLog<V> {
     private final Supplier<Connection> connectionSupplier;
+    private final String logNamespace;
 
-    private SqlitePaxosStateLog(Supplier<Connection> connectionSupplier) {
+    private SqlitePaxosStateLog(Supplier<Connection> connectionSupplier, String logNamespace) {
```

Comment:
Do we need to enforce something about the logNamespace, I don't suspect sqlite likes any odd table names?